code/game/objects/items/devices/traitordevices.dm

```diff
@@ -190,6 +190,7 @@ effective or pretty fucking useless.
 	slot_flags = ITEM_SLOT_BELT
 	attack_verb = list("whipped", "lashed", "disciplined")
 
+	var/equipslot = SLOT_BELT

```

Comment:
You're replacing all of the `ITEM_SLOT_BELT`s with `SLOT_BELT`. Is this intentional?

Reply:
yeah, it breaks otherwise